### DISCUSSION TITLE
News Cache [2023-03-06]

### DISCUSSION THREAD

**Discussion by amyegan** [vercel/community#1713] [Announcements]
### 🚢  Ships

- New [Vercel Monitoring and Logs](https://www.youtube.com/watch?v=oZLjfTi2Y9w)
    - [Monitoring is now available to view traffic and performance data for improved observability](https://vercel.com/changelog/monitoring-is-now-available-to-view-traffic-and-performance-data-for)
    - [Enhanced Logs to search, inspect, and share runtime logs](https://vercel.com/changelog/enhanced-logs-ui-to-search-inspect-and-share-application-logs)
- [Activity date filtering now available](https://vercel.com/changelog/activity-date-filtering-now-available)
- [Add real-time analytics to your project with the Tinybird integration](https://vercel.com/changelog/add-real-time-analytics-to-your-application-with-tinybird-integration)

### 💡 Inspiration

- [Colored Octicon SVGs](https://octicons.andre601.ch/) from @Andre601 :octocat:
- [Vercel Sans](https://twitter.com/evilrabbit_/status/1632742989880606720?s=20) is coming this Spring ✨ 
- [watchthis.dev](http://watchthis.dev) got a [new UI](https://twitter.com/steph_dietz_/status/1631697694363090944?s=20) 💅 

### 📚 Resources

- [Learn how to use Next.js built-in SEO support](https://twitter.com/nextjs/status/1630587612682395648?s=20) with @leerob 
- [Build an AI-personalized media app](https://youtu.be/MY1cuWte_YM) with @StephDietz

**Comment by Andre601** [comment]
Didn't expect to get mentioned. Thanks I guess 😅

**Reply by amyegan** [reply]
I thought it was an awesome project, but I can totally take it off the list if you want!

**Reply by Andre601** [reply]
I didn't say it was bad by any means. Was just surprised to get the honor of being mentioned.

Tho, to be fair, I'm interested in improving the page, as I'm sure I can create a better way to customize the SVGs than what I do right now... Tho I have 0 experience with JS and that stuff...

**Comment by joggienl** [comment]
Cool to see the monitoring available on Pro now!

I have a question about the pricing. The base price is clear, but above 250.000 “metrics” you pay a price per million metrics.

To be able to do some cost calculations upfront, what is a metric? It is unclear now what is counted towards one.

**Reply by amyegan** [reply]
Great question, Jogchum! I asked the Monitoring team for clarification and they said,

> A monitoring metric gets generated whenever one of your projects serves a request or performs server prerendering or incremental static regeneration. Requests will be the dominating factor for most teams

**Reply by joggienl** [reply]
It seems like that means that I should add up the cached and uncached requests from the network tab on the usage dashboard? Is that correct?

**Reply by joggienl** [reply]
I am trying to make a real calculation of what it might mean for a project. If I simply do the calculation as I mentioned, just add the cached and uncached requests I see on the dashboard, it gives me a very interesting number 😬 (very high).

It might help to see an example calculation on the monitoring page where the limits are described.

**Reply by amyegan** [reply]
I just checked with the team and they said that you can use the total requests to get a rough estimate. You can plug it into https://www.wolframalpha.com/input?i=10+%2B+9*+ceil%28%28x-250000%29%2F1000000%29+for+x+from+0+to+10%2C000%2C000 for an estimate of the cost.

They're also working on documentation updates based on your feedback. We really appreciate you pointing out that this wasn't clear so we know how we can improve the docs 🙇

**Reply by joggienl** [reply]
Thank you for checking (again), helpful!

It does (unfortunately 😬) mean my initial calculation based on your earlier comment was correct 🙈 .
